src/rust/ensogl/lib/components/src/list_view/entry.rs

```diff
@@ -308,26 +216,31 @@ impl IdAtYPosition {
 /// Not all entries are displayed at once, only those visible.
 #[derive(Clone,CloneRef,Debug)]
 pub struct List {
```

Comment:
Updae docs of this structure. It is no longer handler for the entries, because it also handles selection.

Reply:
I am not sure what you mean.

Reply:
The docs says
```
/// A view containing an entry list, arranged in column.
///
/// Not all entries are displayed at once, only those visible.
``` 
There is nothing about selection. Once selection is added, the docs should mention it I believe.